docs/public-lectures.md

```diff
@@ -0,0 +1,38 @@
+The OSG User School is our premier training program for researchers,
+facilitators, and others who are interested in learning more about how to
+use High Throughput Computing to advance research.  This year's program is
+online again and the application process is complete.  BUT we have some
+exciting news about sharing the OSG School with the entire community!
+
+**For the first time, the OSG Virtual School 2021 will open its core lectures
+live to the public**
+
+The public lectures are excellent learning opportunities for students and
+researchers who could not be full participants of the School.  The sessions
+are also well suited for those interested in learning more about High
+Throughput Computing, OSG, HTCondor, or leveraging any of these in your
+computing research.  Topics include:
+
+* The main concepts of High Throughput Computing (HTC)
+* The basics of using HTCondor
+* Getting started on OSG, a national distributed HTC infrastructure for
+   Open Science
+* How to prepare software for use in HTC systems
+* Handling data (input and output) in HTC systems
+
+The public lectures are held daily, August 2-6, from 2:30-4:00 pm US
+Central Time.  
+
+The detailed schedule is at: [Lecture Schedule](https://indico.fnal.gov/event/49686/timetable/)
```

Comment:
Let’s try to make this more of a sentence … [The detailed schedule] is available on the Indico site. (Or something like that).